src/link-manager.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @fileoverview Link manager for controlling the link query.
+ */
+
+/** @const */
+genotet.linkManager = {};
+
+/**
+ * Object that stores the link views by their type.
+ */
+/** @type {!Object<!Array<{
+ *    target: genotet.View,
+ *    action: string,
+ *    response: string
+ *  }>>}
+ */
+genotet.linkManager.links = {};
+
+/** @const {!Array<string>} */
+genotet.linkManager.EXPRESSION_ACTIONS = ['addGeneProfile'];
+
+/** @const {!Array<string>} */
+genotet.linkManager.NETWORK_ACTIONS = ['nodeClick', 'edgeClick'];
+
+/** @const {!Array<string>} */
+genotet.linkManager.BINDING_ACTIONS = ['updateTrack', 'locus'];
+
+/**
+ * Initializes the link manager.
+ */
+genotet.linkManager.init = function() {
+  genotet.linkManager.links = {};
+};
+
+
+/**
+ * Set up the link manager.
+ */
+genotet.linkManager.link = function() {
```

Comment:
Please note that message handling can be incremental. Otherwise you can only handle presets rather than future user configs. It's better not to you can create the links once for all.